Frontend/src/database/Database.js

```diff
@@ -116,6 +116,17 @@ class Database {
     );
   }
 
+  async countDocs(selectors) {
```

Comment:
```suggestion
  countDocs(selectors) {
```
No need for async here, since you return a promise anyway.

Reply:
Mhh, I had it before without async but then it didn't work :-/ that's why I added `async` in the first place

now it works, strange...